crates/node/src/lib.rs

```diff
@@ -428,6 +428,37 @@ async fn handle_line(node: &mut Node, line: String) -> eyre::Result<()> {
 
                         println!("{IND} Deleted context {}", context_id);
                     }
+                    "transactions" => {
+                        let Some(context_id) = args else {
+                            println!("{IND} Usage: context transactions <context_id>");
+                            break 'done;
+                        };
+
+                        let Ok(context_id) = context_id.parse() else {
+                            println!("{IND} Invalid context ID: {}", context_id);
+                            break 'done;
+                        };
+
+                        let handle = node.store.handle();
+
+                        let key = calimero_store::key::ContextTransaction::new(
+                            context_id,
+                            [0; 32].into(),
+                        );
+
+                        println!("{IND} {c1:44} | {c2:44}", c1 = "Hash", c2 = "Prior Hash");
+
+                        for (k, v) in &mut handle.iter(&key)?.entries() {
+                            let entry = format!(
+                                "{c1:44} | {c2:?}",
```

Comment:
is it more useful to `Debug` than `Display` the prior hash here?

```suggestion
                                "{c1:44} | {c2}",
```